src/Main.cs

```diff
@@ -226,6 +325,9 @@ private void loadFileBtn_Click(object sender, EventArgs e)
                 {
                     LoadJoinedGuilds();
 
+                    if (task.IsCompleted) LoadTopVC();
+                    else task.ContinueWith(t => LoadTopVC());
```

Comment:
This seems out of place. Why is it basing off the main load task inside the continuation of the guilds load task?

It'd be better if it was just put next to it:
```suggestion
                    LoadJoinedGuilds();
                    LoadTopVC();
```
And then `guildsTask` and `DataPackage.LoadGuilds()` could be renamed to ex. `analyticsTask` and `LoadAnalytics()` to reflect the fact that it's now processing more data than just joined guilds.

Reply:
I did that because the top VC requires data from both, main and analytics tasks.
I moved it below the `task.Start()` with a `Task.WhenAll().ContinueWith()`
```suggestion
                    LoadAnalytics();
                        
                    serversStatusStrip.Visible = false;
                }, TaskScheduler.FromCurrentSynchronizationContext());
                        
                serversStatusStrip.Visible = true;
                        
                task.Start();
                analyticsTask.Start();
                        
                Task.WhenAll(new Task[] { task, analyticsTask }).ContinueWith((_,_)=>LoadTopVC(),null, TaskScheduler.FromCurrentSynchronizationContext());
```